### DISCUSSION TITLE
How to rename typescript files with lsp

### DISCUSSION THREAD

**Discussion by emersonfbarros** [LazyVim/LazyVim#5241] [Q&A]
In the typescript extras there is the configuration for the client command [_typescript.moveToFileRefactoring](https://github.com/LazyVim/LazyVim/blob/d0c366e4d861b848bdc710696d5311dca2c6d540/lua/lazyvim/plugins/extras/lang/typescript.lua#L134C1-L183C24) which from the code you can deduce is intended to rename files consistently on imports using lsp, but how to use this? I don't see any keymap or user command intended for this action in the typescript extras.

**Comment by dpetka2001** [comment]
Probably with code actions by pressing `<leader>ca`? I don't do TS and can't really check it, but that's my assumption.
